fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/TopNType.java

```diff
@@ -0,0 +1,29 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.optimizer.operator;
+
+import com.starrocks.thrift.TTopNType;
+
+public enum TopNType {
+    ROW_NUMBER,
```

Comment:
Does the `ROW_NIUMBER` means `row_number()` window function?

Reply:
Actually, `ROW_NUMBER` represent the ordinary meaning of topn, e.g. limit by number of rows. Since the other two names come from the window function `rank` and `dense_rank`, so I naming it as `row_number` for naming style consistent